packages/react-native-op-sqlite/src/index.ts

```diff
@@ -23,5 +23,7 @@ const opSqlManagerInstance = new ReplicacheGenericSQLiteDatabaseManager(
   genericDatabase,
 );
 
-export const createReplicacheReactNativeOPSQLiteExperimentalCreateKVStore =
-  getCreateReplicacheSQLiteExperimentalCreateKVStore(opSqlManagerInstance);
+export const createReplicacheReactNativeOPSQLiteKVStore = {
+  create: getCreateReplicacheSQLiteKVStore(opSqlManagerInstance),
+  drop: opSqlManagerInstance.destroy,
```

Comment:
@Braden1996 This needs to arrow function or it doesn't bind correctly. 

```suggestion
  drop: (name:string) => opSqlManagerInstance.destroy(name),
```